client/rpcserver/types.go

```diff
@@ -400,6 +409,10 @@ func parseTradeArgs(params *RawParams) (*tradeForm, error) {
 	if err != nil {
 		return nil, err
 	}
+	options, err := checkMapArg(params.Args[8], "options")
```

Comment:
Please update the help doc in handlers.go for `tradeRoute:`.  It can read similarly to the `settings` arg description for `newWalletRoute`.

Reply:
Should be fixed now